internal/platform/database/databasetest/docker.go

```diff
@@ -0,0 +1,76 @@
+package databasetest
+
+import (
+	"bytes"
+	"encoding/json"
+	"os/exec"
+	"testing"
+)
+
+// container tracks information about a docker container started for tests.
+type container struct {
+	ID   string
+	Host string // IP:Port
+}
+
+// startContainer runs a postgres container to execute commands.
+func startContainer(t *testing.T) *container {
```

Comment:
Wonder if we don't pass in the container name as a param?

Reply:
No because the function is hard coded to extract postgres specific information from the container once it is running. We *could* let that function take an image name but it has to be some version of postgres.

This is not a generic "start any kind of container" function.